docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -4,7 +4,8 @@ description: How to configure Teleport to automatically create transient host us
 ---
 
 Teleport's SSH Service can be configured to automatically create local Unix users
-upon login.
+upon login.  Refer to the [Desktop Access RBAC guide](../desktop-access/rbac.mdx#automatic-user-creation)
```

Comment:
I would place the new sentence as a separate paragraph at the end of the introductory section, rather than as the second sentence in the first paragraph. For users who are reading the correct guide (i.e., who want to set up host user creation for SSH), the new sentence gets in the way a bit.